src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/DocumentExtensions.cs

```diff
@@ -217,17 +212,5 @@ public static async ValueTask<AnalyzerConfigOptions> GetAnalyzerConfigOptionsAsy
             return document.Project.AnalyzerOptions.AnalyzerConfigOptionsProvider.GetOptions(syntaxTree);
         }
 #endif
-
-        public static SyntaxEditor GetSyntaxEditor(this Document document, SyntaxNode root)
```

Comment:
removed this extension which just existed so we could customize CODE_STYLE behavior prior to these types becoming public.